File: linguagem_dart/classes_e_objetos/hospital/medico.dart
import 'hospital.dart';

class Medico {
  // Atributos da classe Medico
  String nome = "";
  late Hospital hospital;

  // Construtor da classe Medico
  Medico(this.nome, this.hospital) {}

  String atender() {
    return "O(a) médico(a) $nome está atendendo no ${hospital.nome}";
  }
}

File: linguagem_java/classes_e_objetos/hospital/Medico.java
package classes_e_objetos.hospital;

public class Medico {
    // Atributos da classe Médico
    String nome;
    Hospital hospital;

    // Construtor da classe Médico
    Medico(String nome, Hospital hospital) {
        this.nome = nome;
        this.hospital = hospital;
    }

    String atender() { 
        return "O(a) médico(a) " + nome + " está atendendo no " + hospital.nome;
    }
}
